cogboard-app/src/main/kotlin/com/cognifide/cogboard/logStorage/QuarantineRule.kt

```diff
@@ -0,0 +1,37 @@
+package main.kotlin.com.cognifide.cogboard.logStorage
+
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+
+data class QuarantineRule(
+    val label: String,
+    val reasonField: String,
+    val regex: Regex,
+    val enabled: Boolean
+) {
+    companion object {
+        private const val LABEL = "label"
+        private const val REASON = "reasonField"
+        private const val REGEX = "regExp"
+        private const val ENABLED = "checked"
+
+        fun from(json: JsonObject): QuarantineRule? {
+            return try {
+                QuarantineRule(
+                        json.getString(LABEL)!!,
+                        json.getString(REASON)!!,
+                        json.getString(REGEX)!!.toRegex(),
+                        json.getBoolean(ENABLED)!!
+                )
+            } catch (_: NullPointerException) {
+                null
+            }
+        }
```

Comment:
I would rather return not null objects in such cases. Could we have a default rule that does noting?

Reply:
Done.